app/routes/users/components/UserProfile.js

```diff
@@ -312,6 +312,66 @@ export default class UserProfile extends Component<Props, EventsProps> {
                 </Card>
               </div>
             )}
+            {/* canChangeGrade is a good heuristic if we should show permissions. All users can see their own permission via the API, but only admins can show permissions for other users.*/}
+            {canChangeGrade && (
+              <div>
+                <h3>Rettigheter</h3>
+                <Card className={styles.infoCard}>
+                  {uniqBy(
+                    nestedPermissions.concat(
+                      // $FlowFixMe flatMap is polyfilled
+                      nestedPermissions.flatMap(
+                        ({ parentGroups }) => parentGroups
+                      )
+                    ),
+                    a => a.abakusGroup.id
+                  ).map(
+                    ({ abakusGroup, permissions }) =>
+                      !!permissions.length && (
+                        <>
+                          <h4>
+                            Rettigheter fra
+                            <Link
+                              to={`/admin/groups/${
+                                abakusGroup.id
+                              }/permissions/`}
+                            >
+                              {' '}
+                              {abakusGroup.name}{' '}
+                            </Link>
+                          </h4>
+                          <ul>
+                            {permissions.map(permission => (
+                              <li key={permission + abakusGroup}>
```

Comment:
string + object 🙃 
```
const stuff = "perm" + {"id": 1, "name": "name"};
undefined
suff
"[object Object]perm"
```

Guess I'll JS then 👍

Reply:
If the object always gets parsed to _"[object Object]"_  will just be the same as setting key `key={permission}`?

Reply:
or set it as _abakusGroup.id_ or _abakusGroup.name_ ?

Reply:
Good catch! Used `abakusGroup = pk` at first, but forgot to change this. Thx